src/transactions/feed/detailContent/SwapContent.tsx

```diff
@@ -0,0 +1,133 @@
+    ?.symbol
+
+  //TODO: verifying fee part.
+  const onPressTxDetails = () => {
+    navigate(Screens.WebViewScreen, {
+      uri: `${networkConfig.celoExplorerBaseTxUrl}${exchange.transactionHash}`,
+    })
+  }
+
+  return (
+    <View style={styles.contentContainer}>
+      <View style={[styles.row, { paddingBottom: Spacing.Regular16 }]}>
+        <Text style={styles.exchangeBodyText}>{t('swapContent.swapTo')}</Text>
+        <TokenDisplay
+          style={styles.currencyAmountText}
+          amount={exchange.outAmount.value}
+          tokenAddress={exchange.outAmount.tokenAddress}
+          showLocalAmount={false}
+          showSymbol={true}
+          hideSign={true}
+          testID="SwapContent/swapFrom"
+        />
+      </View>
+      <View style={[styles.row, { paddingBottom: Spacing.Regular16 }]}>
+        <Text style={styles.exchangeBodyText}>{t('swapContent.swapFrom')}</Text>
+        <TokenDisplay
+          style={styles.currencyAmountText}
+          amount={exchange.inAmount.value}
+          tokenAddress={exchange.inAmount.tokenAddress}
+          showLocalAmount={false}
+          showSymbol={true}
+          showExplicitPositiveSign={true}
+          hideSign={true}
+          testID="SwapContent/swapTo"
+        />
+      </View>
+      <View style={styles.separator} />
+      <View style={[styles.row, { paddingBottom: Spacing.Smallest8 }]}>
+        <Text style={styles.exchangeBodyText}>{t('swapContent.rate')}</Text>
+        <Text testID="SwapContent/rate" style={styles.currencyAmountText}>
+          {`1 ${fromTokenSymbol} ≈ ${formatValueToDisplay(
+            new BigNumber(exchange.inAmount.value).dividedBy(exchange.outAmount.value)
+          )} ${toTokenSymbol}`}
+        </Text>
+      </View>
+      <View style={[styles.row, { paddingBottom: Spacing.Smallest8 }]}>
+        <Text style={styles.exchangeBodyText}>{t('swapContent.estimatedFee')}</Text>
+        <TokenDisplay
+          style={styles.currencyAmountText}
+          amount={exchange.fees[0].amount.value}
+          tokenAddress={exchange.fees[0].amount.tokenAddress}
+          showLocalAmount={false}
+          showSymbol={true}
+          hideSign={true}
+          testID="SwapContent/estimatedFee"
+        />
+      </View>
+      <Touchable testID={'txDetails'} borderless={true} onPress={onPressTxDetails}>
+        <View style={styles.rowContainer}>
+          <Text style={styles.txDetails}>
+            {t('fiatConnectStatusScreen.withdraw.success.txDetails')}
+          </Text>
+          <OpenLinkIcon color={colors.gray4} />
+        </View>
+      </Touchable>
+    </View>
+  )
+}
+
+const styles = StyleSheet.create({
+  contentContainer: {
+    flexShrink: 1,
+  },
+  row: {
+    flex: 1,
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+  },
+  exchangeBodyText: {
+    ...fontStyles.large,
+    width: '40%',
+    textAlign: 'left',
```

Comment:
is this style needed? i think left is the default

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/2752/commits/22c9b2cad705288660aa858ccbf1e2e069ccafb8!